### DISCUSSION TITLE
Duda ejercicio tablas de hash

### DISCUSSION THREAD

**Discussion by IgnacioHqz** [IIC2133-PUC/2024-2#183] [Preguntas Materia]
Hola, sé que puede ser muy encima que esté preguntando, pero no he logrado entender una resolución que se hace a un ejercicio de una I2 del semestre pasado. El ejercicio es el que adjunto en la foto, y mi duda es que por qué utilizan el DV y luego la segunda función de hash es RUNmod7 para guardar el registro? con el módulo 7 infiero que el tamaño de la tabla es de 7, pero cómo se guarda cada dato (los puntajes de la PAES) del registro en una celda a partir del RUT? Si me pudieran explicar por favor ya que no logro entender la idea de la respuesta. 
Gracias!

![image](https://github.com/user-attachments/assets/83b491d8-fafc-4a23-9692-cfb52f38ccba)
![image](https://github.com/user-attachments/assets/2f5810c1-ce7f-43fd-9811-1c8282703cf6)

**Comment by tomastrivino** [comment]
Perdona! Sí fue algo encima. Si mantienes la duda háznoslo saber.
